src/main/java/be/ephec/java/exercice/Main.java

```diff
@@ -1,7 +1,51 @@
 package be.ephec.java.exercice;
```

Comment:
Je vous conseille de répondre aux questions dans les packages correspondants des exercices...

Reply:
bien noté !